neuromation/cli/config.py

```diff
@@ -203,24 +203,75 @@ def config() -> None:
 @async_cmd()
 async def get_clusters(root: Root) -> None:
     """
-    Fetch and display the list of available clusters from the Neuro Platform.
+    Fetch and display the list of available clusters.
 
     """
-
-    config = await get_server_config(root.client._session, root.url, root.auth)
+    click.secho("Fetch the list of available clusters...", dim=True)
+    await root.client.config.fetch()
     fmt = ClustersFormatter()
     pager_maybe(
-        fmt(config.clusters, config.cluster_config.name), root.tty, root.terminal_size
+        fmt(root.client.config.clusters.values(), root.client.config.cluster_name),
+        root.tty,
+        root.terminal_size,
     )
 
 
+@command()
+@click.argument("cluster_name", required=False, default=None, type=str)
+@async_cmd()
+async def switch_cluster(root: Root, cluster_name: Optional[str]) -> None:
+    """Switch the active cluster.
+
+    CLUSTER_NAME is the cluster name to select.  The interactive prompt is used if the
+    name is omitted (default).
+
+    """
+    click.secho("Fetch the list of available clusters...", dim=True)
+    await root.client.config.fetch()
+    if cluster_name is None:
+        if not root.tty:
+            raise click.BadArgumentUsage(
+                "Interactive mode is disabled for non-TTY mode, "
+                "please specify the CLUSTER_NAME"
+            )
+        real_cluster_name = await prompt_cluster(root.client)
+    else:
+        real_cluster_name = cluster_name
+    await root.client.config.switch_cluster(real_cluster_name)
+    click.echo(
+        "The current cluster is " + click.style(real_cluster_name, underline=True)
+    )
+
+
+async def prompt_cluster(client: Client) -> str:
+    clusters = client.config.clusters
+    while True:
+        fmt = ClustersFormatter()
+        click.echo("\n".join(fmt(clusters.values(), client.config.cluster_name)))
+        answer = input(f"Select cluster to switch [{client.config.cluster_name}]: ")
```

Comment:
if a user hits return right away, should we fall back to `[{client.config.cluster_name}]`?